controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -445,9 +447,12 @@ private DeploymentMetrics deploymentMetricsFromSlime(Inspector object) {
 
     private RotationStatus rotationStatusFromSlime(Inspector parentObject) {
         var object = parentObject.field(rotationStatusField);
-        var statusMap = new LinkedHashMap<RotationId, Map<ZoneId, RotationState>>();
+        var statusMap = new LinkedHashMap<RotationId, RotationStatus.Targets>();
         object.traverse((ArrayTraverser) (idx, statusObject) -> statusMap.put(new RotationId(statusObject.field(rotationIdField).asString()),
-                                                                              singleRotationStatusFromSlime(statusObject.field(statusField))));
+                                                                              new RotationStatus.Targets(
+                                                                                      singleRotationStatusFromSlime(statusObject.field(statusField)),
+                                                                                      // TODO(mpolden): Make non-optional after October 2019
+                                                                                      Serializers.optionalInstant(statusObject.field(lastUpdatedField)).orElse(Instant.EPOCH))));
```

Comment:
Could just make an instance from the field, without optionality — non-existent value is 0, which is EPOCH.